CHANGELOG.rst

```diff
@@ -1,5 +1,27 @@
 .. _changelog:
 
+0.7.3
+-----
+
+This is a minor release that contains mainly improvements and fixes to the Renku CLI.
+Templates now use this new Renku CLI version ``0.12.2`` by default.
+
+New features
+~~~~~~~~~~~~
+
+- **Migrations** A ``migrationscheck`` command has been added.
```

Comment:
User's won't know what this means - we can say something like "Projects using older versions of renku can be automatically migrated."

Reply:
We had project migration before.
I think this is actually a new command that shows the state of a project, a user could run this inside an environment or locally to check the state. [ref issue](https://github.com/SwissDataScienceCenter/renku-python/pull/1761)

Reply:
I've changed this part in the new release notes